packages/office-ui-fabric-react/src/components/pickers/AutoFill/BaseAutoFill.Props.ts

```diff
@@ -59,4 +59,26 @@ export interface IBaseAutoFillProps extends React.HTMLProps<HTMLInputElement | B
    */
   enableAutoFillOnKeyPress?: KeyCodes[];
 
+  /**
+   * the default value to be visible
+   */
+  defaultVisibleValue?: string;
+
+  /**
+   * Handler for checking the next props in componentWillReceiveProps
+   * and updating the value if needed
+   *
+   * @param {IBaseAutoFillProps} nextProps - the props that got passed in to the auto fill's componentWillReceiveProps
+   * @returns {string} - the updated value to set, if needed
+   */
+  onComponentWillReceiveProps?: (nextProps: IBaseAutoFillProps) => string;
```

Comment:
You're right, we already have the props and we don't need to pass them back... removed the parameter and renamed the callbacks